manifests/repo.pp

```
@@ -3,7 +3,7 @@
   anchor { 'jenkins::repo::alpha': }
   anchor { 'jenkins::repo::omega': }
   case $::operatingsystem {
-    centos, redhat, oel, Amazon: {
+    centos, redhat, oel, Amazon, fedora: {
```

I was not aware of this fact and usually just copied the widely used operatingsystem pattern, but osfamily sounds much saner. I did notice that you need a relative recent facter version (1.6.2) because some servers that weren't fully updated (I know, bad me) were missing the fact.

I'd also argue for a default: { fail("Unsupported OS family") }